src/promptflow/setup.py

```diff
@@ -98,11 +98,15 @@
             # MDC dependencies for monitoring
             "azureml-ai-monitoring>=0.1.0b3,<1.0.0",
         ],
+        "executor-service": [
+            "uvicorn>=0.27.0",  # used to start executor service
```

Comment:
both uvicorn and fastapi are still in beta Development Status. We might avoid dep on such libraries.
![image](https://github.com/microsoft/promptflow/assets/47586720/e31f6c86-d616-4861-8b54-06294d49e5aa)